modules/ROOT/pages/getting-started.adoc

```diff
@@ -475,7 +475,12 @@ The configuration file and logs are located in `/Users/sync_gateway`.
 
 The following steps explain how to connect Sync Gateway to the Couchbase Server instance.
 
-* Open a new file called *sync-gateway-config.json* with the following.
+* You must first create a Sync Gateway *database*.
+A Sync Gateway database is a namespace for documents and multiple databases can be defined in the configuration file for Sync Gateway.
+Each Sync Gateway *database* connects to a Couchbase Server *bucket*.
```

Comment:
A Sync Gateway can be configured with several databases, each of which points to a Couchbase Server bucket.

Reply:
@rajagp does this still need changing?